.github/workflows/integration.yml

```diff
@@ -178,7 +178,7 @@ jobs:
     - name: Set up Node.js
       uses: actions/setup-node@v1
```

Comment:
There's also one more change to make here. I also updated `actions/setup-node` to use `v2` instead of `v1` for the other OS CI in #1714

Reply:
Thanks for catching that!